force-app/main/default/classes/TestAccountController.cls

```diff
@@ -18,7 +18,7 @@ public class TestAccountController {
         List<Account> accts = AccountController.getAccountList();
         Test.stopTest();
 
-        System.assertEquals(accts.size(), 10);
+        Assert.areEqual(accts.size(), 10);
```

Comment:
@satyasekharcvb can you fix the order of all asserts parameters: it's expected then, actual.
```suggestion
        Assert.areEqual(10, accts.size());
```

Reply:
Thank you for pointing this @pozil! I have fixed it wherever necessary.